.circleci/config.yml

```
@@ -34,7 +34,7 @@ jobs:
           command: |
             python3.8 -m venv venv/
             venv/bin/pip install pip-tools --constraint requirements.in
-            venv/bin/pip-sync
+            venv/bin/pip-sync --pip-args=--no-deps
```

By default I think pip-sync only looks for `requirements.txt`, so should this specify both `requirements.txt` and `java-requirements.txt` like is done elsewhere?
```suggestion
            venv/bin/pip-sync --pip-args=--no-deps requirements.txt \
              java-requirements.txt
```

---

java requirements are intentionally left out here, as they are optional and should not be required for unit tests.